### DISCUSSION TITLE
posts/writing-clean-tuples-in-scala/

### DISCUSSION THREAD

**Discussion by giscus** [AvaPL/avapl.github.io#4] [Announcements]
# posts/writing-clean-tuples-in-scala/

How many times have you wondered what is hidden under ._1 in a line of code? How many filter conditions were not clear because they relied on tuple accessors? Using tuples is quite easy, although it creates many opportunities to write something nasty. This post contains tips that will help you improve the readability of your tuples.

https://avapl.github.io/posts/writing-clean-tuples-in-scala/

**Comment by AvaPL** [comment]
Hi! Thank you for reading. As this is my first post, any feedback is welcome.
